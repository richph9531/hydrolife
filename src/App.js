
import PageHeader from './components/PageHeader';

import classes from './styles/App.module.css';

function App() {
  return (
    <div className='App'>
      <PageHeader/>
      <div className={classes.MainContainer}>
        <img 
          src={`${process.env.PUBLIC_URL}/media/logo.png`}
          alt='Hydrolife Logo'/>
        <p>Coming soon...</p>
        <a href="tel:0878203953" style={{ color: '#FFF' }}>087 820 3953</a>
      </div>
    </div>
  );
}

export default App;
