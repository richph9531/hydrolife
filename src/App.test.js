import { render, screen } from '@testing-library/react';
import App from './App';

test('Page DOM is correct', () => {
  const view = render(<App />);
  expect(view.baseElement).toMatchSnapshot();
});
