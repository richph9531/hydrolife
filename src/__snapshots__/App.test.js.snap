// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page DOM is correct 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <div
        class="PageHeader"
      >
        <div
          class="LogoContainer"
        >
          <img
            alt="Hydrolife Logo"
            src="/media/logo.png"
          />
        </div>
      </div>
      <div
        class="MainContainer"
      >
        <img
          alt="Hydrolife Logo"
          src="/media/logo.png"
        />
        <p>
          Coming soon...
        </p>
        <a
          href="tel:0878203953"
          style="color: rgb(255, 255, 255);"
        >
          087 820 3953
        </a>
      </div>
    </div>
  </div>
</body>
`;
